2/contrib/jenkins/jenkins-common.sh

```diff
@@ -16,6 +16,9 @@ function generate_passwd_file() {
 
   if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"997" ]; then
 
+    # strip out the value that CRIO injected.
+    grep -v $USER_ID:x:$USER_ID /etc/passwd > /tmp/passwd
```

Comment:
this grep is based on how crio injects the entry:
https://github.com/cri-o/cri-o/pull/2022/files#diff-cd575165f84eed10d10eb1460b0927b4R261